mac/Keyman4Mac/Keyman4Mac/Frameworks/KeymanEngine4Mac.framework/Headers

```diff
@@ -0,0 +1 @@
+Versions/Current/Headers
```

Comment:
So we can either: a) Add a build step to the test app to copy over the framework so it can be linked in without being checked into the repo; OR b) Remove the Framework folder from the repo and try to figure out the source of the warnings. Choice "a" feels like the right thing to do. That would mean we either need to add the missing header files (KMCompStore.h & KCompGroup.h) to those made public by the engine (assuming that is safe/desirable) or remove the little bit of code (half of which is debug-only) that spews out NSLog messages for the stores and groups in the selected Keyboard.